NEWS.md

```diff
@@ -1,3 +1,66 @@
+# New in snapd 2.64:
+* Support building snapd using base Core22 (Snapcraft 8.x)
+* FIPS: support building FIPS complaint snapd variant that switches to FIPS mode when the system boots with FIPS enabled
+* AppArmor: update to AppArmor 4.0.1
+* AppArmor: support AppArmor snippet priorities
+* AppArmor prompting: add path pattern matching
+* AppArmor prompting: support constraints and abstract permissions
+* Registry views (experimental): rename from aspects to registries
+* Registry views (experimental): support reading registry views using snapctl
+* Registry views (experimental): restrict view paths from using a number as first character and view names to storage path style patterns
+* Snap components: support installing snaps and components from files at the same time (no REST API/CLI)
+* Snap components: support downloading components related assertions from the store
+* Snap components: support installing components from the store (no REST API/CLI)
+* Snap components: support removing components (REST API, no CLI)
+* Snap components: started support for component hooks
+* Kernel modules as components: support kernel modules as components
+* Refresh app awareness (experimental): add data field for /v2/changes REST API to allow associating each task with affected snaps 
+* Refresh app awareness (experimental): use the app name from .desktop file in notifications
+* Allow re-exec when SNAP_REEXEC is set for unlisted distros to simplify testing
+* Generate GNU build ID for Go binaries
+* Add missing etelpmoc.sh for shell completion
+* Do not attempt to run snapd on classic when re-exec is disabled
+* Packaging/build maintenance for Debian sid, Fedora, Arch, openSuse
+* Add snap debug api command to enable running raw queries
+* Enable snap-confine snap mount directory detection
+* Replace global seccomp filter with deny rules in standard seccomp template
+* Remove support for Ubuntu Core Launcher (superseded by snap-confine)
+* Support creating pending serial bound users after serial assertion becomes available
+* Support disabling cloud-init using kernel command-line
+* Split refresh into essential and non-essential snaps to speed up and minimise snapd restarts
```

Comment:
This is true but makes more sense for us internally. Maybe this is more understandable for other people:
```suggestion
* In hybrid systems, apps can refresh without waiting for restarts required by essential snaps
```
It omits many caveats but I think it's more appropriate. wdyt?

Reply:
+1 for this reading better for people who are not snapd devs

Reply:
It was an attempt to combine all of the below which all seems related to split refresh. Perhaps we need more than one entry to represent it?

<google-sheets-html-origin><!--td {border: 1px solid #cccccc;}br {mso-data-placement:same-cell;}-->
14052 | https://github.com/snapcore/snapd/pull/14052 | o/snapstate: refactor/optimise dependency setup in split refresh
-- | -- | --
14039 | https://github.com/snapcore/snapd/pull/14039 | o/snapstate: enable split refresh for auto-refresh
13991 | https://github.com/snapcore/snapd/pull/13991 | o/snapstate: don't wait for def provider if it depends on base in split refresh
13955 | https://github.com/snapcore/snapd/pull/13955 | o/snapstate: in hybrid systems re-refresh before reboot if possible
13852 | https://github.com/snapcore/snapd/pull/13852 | o/snapstate: don't make apps wait for reboot on classic w/ kernel

Reply:
fixed